src/database/models/role.js

```diff
@@ -0,0 +1,22 @@
+const { Model } = require('sequelize');
+
+module.exports = (sequelize, DataTypes) => {
```

Comment:
Change this to ES6, please. Refer to previous comment on database/models/index.js